packages/gatsby-theme-newrelic/src/components/Callout.js

```diff
@@ -34,6 +52,7 @@ const Callout = ({ title, variant, children }) => {
       css={css`
         padding: 1.25rem;
         margin: 1.5rem 0;
+        border-radius: 3px;
```

Comment:
Could we use `0.25rem`? That border radius is pretty standard across our components with a border radius.